### DISCUSSION TITLE
Error with landsatxplore API

### DISCUSSION THREAD

**Discussion by deepikas2010** [yannforget/landsatxplore#56] [Q&A]
Hi,

The code was working before without any issue but I'm getting the below error today. Could you please help me to resolve this issue?
    api = landsatxplore.api.API(username,password)
  File "/home/deepika/.local/lib/python3.6/site-packages/landsatxplore/api.py", line 28, in __init__
    self.key = self.login(username, password)
  File "/home/deepika/.local/lib/python3.6/site-packages/landsatxplore/api.py", line 48, in login
    response = requests.post(self.endpoint + 'login?', data=data).json()
  File "/home/deepika/.local/lib/python3.6/site-packages/requests/models.py", line 897, in json
    return complexjson.loads(self.text, **kwargs)
  File "/usr/lib/python3/dist-packages/simplejson/__init__.py", line 518, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python3/dist-packages/simplejson/decoder.py", line 370, in decode
    obj, end = self.raw_decode(s)
  File "/usr/lib/python3/dist-packages/simplejson/decoder.py", line 400, in raw_decode
    return self.scan_once(s, idx=_w(s, idx).end())
simplejson.errors.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

**Comment by MuhammadAsim1214** [comment]
I am also getting the same error. Have you solved it?

**Reply by deepikas2010** [reply]
Hi, 

I couldn't resolve it. I'm still facing the issue.

**Comment by MuhammadAsim1214** [comment]
Try download_landsat_scenes.py.

**Comment by MuhammadAsim1214** [comment]
https://edclpdsftp.cr.usgs.gov › download_landsat_scenes
<https://edclpdsftp.cr.usgs.gov/downloads/acix2scenes/download_landsat_scenes.py>
<https://www.google.com/search?q=https%3A%2F%2Fedclpdsftp.cr.usgs.gov+%E2%80%BA+download_landsat_scenes&rlz=1C1NHXL_noNO838NO838&oq=https%3A%2F%2Fedclpdsftp.cr.usgs.gov+%E2%80%BA+download_landsat_scenes&aqs=chrome..69i57j69i58&sourceid=chrome&ie=UTF-8#>

   1.
   <https://webcache.googleusercontent.com/search?q=cache:I0v-449iwe8J:https://edclpdsftp.cr.usgs.gov/downloads/acix2scenes/download_landsat_scenes.py+&cd=1&hl=en&ct=clnk&gl=no>

**Reply by deepikas2010** [reply]
Hi Asim,

Thank you so much. I will try the suggested code.

**Reply by deepikas2010** [reply]
I have tried the above suggested code. There is no issue with my username and password but it didn't give the expected results.
I'm looking for available search dates without downloading the scenes.
